problem_builder/public/js/data_export.js

```diff
@@ -42,49 +61,76 @@ function DataExportBlock(runtime, element) {
         if (status.last_export_result) {
             if (status.last_export_result.error) {
                 $statusArea.append($('<p>').text(
-                    'Data export failed. Reason: ' + status.last_export_result.error
+                    _.template(
+                        gettext('Data export failed. Reason: <%= error %>'),
+                        {'error': status.last_export_result.error}
+                    )
                 ));
             } else {
                 startTime = new Date(status.last_export_result.start_timestamp * 1000);
                 $statusArea.append($('<p>').text(
-                    'A report is available for download.'
+                    gettext('A report is available for download.')
                 ));
                 $statusArea.append($('<p>').text(
-                    'It was created at ' + startTime.toString() +
-                    ' and took ' + status.last_export_result.generation_time_s.toFixed(1) +
-                    ' seconds to finish.'
+                    _.template(
+                        ngettext(
+                            'It was created at <%= creation_time %> and took <%= seconds %> second to finish.',
+                            'It was created at <%= creation_time %> and took <%= seconds %> seconds to finish.',
+                            status.last_export_result.generation_time_s.toFixed(1)
+                        ),
+                        {
+                            'creation_time': startTime.toString(),
+                            'seconds': status.last_export_result.generation_time_s.toFixed(1)
+                        }
+                    )
                 ));
             }
         } else {
             if (status.export_pending) {
                 $statusArea.append($('<p>').text(
-                    'The report is currently being generated…'
+                    gettext('The report is currently being generated…')
```

Comment:
Nit: edX generally prefers not to include ellipses in messages, I think because they can be hard to translate.